include/cinder/GeomIo.h

```diff
@@ -210,8 +210,12 @@ class Rect : public Source {
 	Rect( const Rectf &r );
 
 	Rect&		rect( const Rectf &r );
+	//! Enables default colors. Disabled by default.
+	Rect&		colors();
```

Comment:
It looks like we handle this differently in different places, however I think the best way is to pass a bool in, i.e. `colors( bool enable = true )`, instead of having hte addition `disableColors()` method. This is how it is done in [Icosehedron](https://github.com/cinder/Cinder/blob/glNext/include/cinder/GeomIo.h#L269), but obviously not in [Cube](https://github.com/cinder/Cinder/blob/glNext/include/cinder/GeomIo.h#L237-L241).

Above all, I think consistency is important so we should probably pick one or the other and make them all match.